### PR TITLE
Feature/create-tier--add-admin-ui-project

### PR DESCRIPTION
**Pull request by JLSRKonk**

### CODE REVIEW

File: AdminUI/admin-ui-app/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# AdminUiApp
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 15.2.0.
```
Comment (tnotheis) [inline AdminUI/admin-ui-app/README.md:3]: Remove this line. It's not interesting for the reader

**Review by tnotheis — CHANGES_REQUESTED** [review]
Shouldn't you wait merging this PR until the Admin API is merged?

File: AdminUI/Structure.txt
[the remark this answers is not in the document]
Comment (JLSRKonk) [reply AdminUI/Structure.txt:1]: This file would contain the entire file structure of the project.
At the time of the project creation, both this file and the README.txt seemed that they could become helpful, but right now it seems there would be no use for them, so I removed them.

File: AdminUI/README.txt
[the remark this answers is not in the document]
Comment (JLSRKonk) [reply AdminUI/README.txt:1]: See the comment to the structure file.

File: AdminUI/admin-ui-app/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Editor configuration, see https://editorconfig.org
+root = true
+
+[*]
+charset = utf-8
+indent_style = space
+indent_size = 2
```
[the remark this answers is not in the document]
Comment (JLSRKonk) [reply AdminUI/admin-ui-app/.editorconfig]: Some of these "issues" are the result of default settings from the angular project. I fixed the ones you mentioned in the latest commits.

File: AdminUI/admin-ui-app/README.md (hunk 1 above)
Comment (JLSRKonk) [reply AdminUI/admin-ui-app/README.md]: This file is also generated by default by angular. I have removed the lines in question.

File: AdminUI/admin-ui-app/src/app/components/quotas-component/identity-component/identity-list/identity-list.component.css
[the remark this answers is not in the document]
Comment (JLSRKonk) [reply AdminUI/admin-ui-app/src/app/components/quotas-component/identity-component/identity-list/identity-list.component.css:1]: With the current naming scheme on the project, the identity-list.component would be used to display the list of identities, while a new component called identity-details.component or identity-edit.component would display a detailed view of an Identity, so there should be no naming conflict. There was some confusion on having a '-component' suffix on the folders that contained one or more components, see the 'quotas-component' folder, so I removed the suffix from all the folders. The actual component files still keep their .component extension.

**Review by daniel-almeida-konkconsulting — APPROVED** [review]

**Pull request merged**